internal/pkg/skuba/kubernetes/jobs.go

```diff
@@ -45,10 +45,16 @@ func DeleteJob(client clientset.Interface, name string) error {
 
 func CreateAndWaitForJob(client clientset.Interface, name string, spec batchv1.JobSpec) error {
 	_, err := CreateJob(client, name, spec)
-	defer DeleteJob(client, name)
 	if err != nil {
 		return err
 	}
+	defer func() {
```

Comment:
@ereslibre reverted the behaviour to the original one, just informing only if there are errors